doc/source/deployment/deploy.rst

```diff
@@ -58,6 +58,35 @@ To check the deployment progress of the Openstack services:
 
   kubectl get po -n openstack
 
+OpenStack Services
+++++++++++++++++++
+
+SUSE Containerized OpenStack deploys OpenStack Cinder, Glance, Heat, Horizon,
+Keystone, Neutron, and Nova.
+
+SUSE Containerized OpenStack deployment automatically adds the following host
+rules to the /etc/hosts file on the deployer as in the following example:
```

Comment:
`[...] adds the following host rules [...] as in the following example:` is kinda clunky and makes it hard to tell that the IP addresses are example addresses. I think we should remove the first occurrence of "the following" to give us `adds host rules [...] as in the following example:`

Reply:
fixed